db/migrate/20221122151902_create_likes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] The references should point to an `author`, not `user`. You can use the `to_table` option to point the database to the User table.

Reply:
Thank you for the review @DammyShittu